apps/emqx_management/src/emqx_mgmt_api_subscriptions.erl

```diff
@@ -168,16 +172,12 @@ subscriptions(get, #{query_string := QString}) ->
             {200, Result}
     end.
 
-format(Items) when is_list(Items) ->
-    [format(Item) || Item <- Items];
-format({{Subscriber, Topic}, Options}) ->
-    format({Subscriber, Topic, Options});
-format({_Subscriber, Topic, Options}) ->
+format(WhichNode, {{_Subscriber, Topic}, Options}) ->
     maps:merge(
         #{
             topic => get_topic(Topic, Options),
             clientid => maps:get(subid, Options),
-            node => node()
+            node => WhichNode
```

Comment:
This seems like a new behavior; maybe add a test?

Reply:
Done. I think it can be covered by `emqx_mgmt_api_SUITE:t_cluster_query/1` now